src/Serializer/AbstractCollectionNormalizer.php

```diff
@@ -54,11 +57,28 @@ public function supportsNormalization(mixed $data, string $format = null, array
         return static::FORMAT === $format && is_iterable($data);
     }
 
+    /**
+     * @deprecated since Symfony 6.3, use "getSupportedTypes(?string $format)" instead
+     */
     public function hasCacheableSupportsMethod(): bool
     {
+        trigger_deprecation('symfony/serializer', '6.3', 'The "%s()" method is deprecated, use "getSupportedTypes()" instead.', __METHOD__);
```

Comment:
```suggestion
        trigger_deprecation('api-platform/core', '3.2', 'The "%s()" method is deprecated, use "getSupportedTypes()" instead.', __METHOD__);
```
Same for all notices (using the right package name) as this is API Platform's code